libs/datalayout/src/main/java/com/linkedin/openhouse/datalayout/persistence/Utils.java

```diff
@@ -0,0 +1,42 @@
+package com.linkedin.openhouse.datalayout.persistence;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.linkedin.openhouse.datalayout.layoutselection.DataLayoutOptimizationStrategy;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.commons.lang.StringEscapeUtils;
+import org.apache.spark.sql.SparkSession;
+
+/** Utility class for strategies persistence. */
+public final class Utils {
+  public static final String DATA_LAYOUT_PROPERTY_KEY = "data-layout-strategies";
+
+  private Utils() {}
+
+  public static void saveStrategies(
```

Comment:
Is this meant to be used for test purposes only? 
I'm not clear on how the strategy generation and saving fits in DLO/compaction flow. Can you add some details pls?